core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review.directive.html

```diff
@@ -65,6 +74,9 @@ <h3>Review translation</h3>
   .oppia-suggestion-resolve-button {
     min-width: 165px;
   }
+  .oppia-translation-review-edit-buttons {
```

Comment:
Can we give this some spacing on the left?

Reply:
Done